schema/dataKeys.json

```diff
@@ -0,0 +1,286 @@
+        },
+        "configMapName": {
+          "type": "string",
+          "description": "The configmap that exists on the cluster"
+        }
+      }
+    },
+    "releaseNotes": {
+      "type": "object",
+      "properties": {
+        "product_id": {
+          "type": "integer",
+          "description": "The product ID e.g 321"
+        },
+        "product_name": {
+          "type": "string",
+          "description": "The product name e.g. exampleproduct"
+        },
+        "product_version": {
+          "type": "string",
+          "description": "The product version e.g v1.0.0",
+          "pattern": "^(fbc|(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)|(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)|(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*) ([Aa]lpha|[Bb]eta|fast|tech preview)|(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*) ([Aa]lpha|[Bb]eta|fast|tech preview))$"
+        },
+        "product_stream": {
+          "type": "string",
+          "description": "The product stream e.g. RHEL-tp1"
+        },
+        "cpe": {
+          "type": "string",
+          "description": "The product CPE ID e.g. cpe:/a:example:openstack:el8"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["RHEA", "RHBA", "RHSA"],
+          "description": "Type advisory type e.g. RHSA"
+        },
+        "synopsis": {
+          "type": "string",
+          "description": "The advisory synopsis e.g. my advisory synopsis"
+        },
+        "topic": {
+          "type": "string",
+          "description": "The advisory topic e.g. my advisory topic"
+        },
+        "description": {
+          "type": "string",
+          "description": "The advisory description e.g. This advisory is for security fixes to my product"
+        },
+        "solution": {
+          "type": "string",
+          "description": "The advisory solution e.g. For details on how to apply this update, see docs.com"
+        },
+        "reference": {
+          "type": "array",
+          "description": "The advisory references e.g. https://access.redhat.com/security/updates/classification & https://docs.redhat.com/some/example/release-notes ",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "sign": {
+      "type": "object",
+      "properties": {
+        "cosignSecretName": {
+          "type": "string",
+          "description": "Name of secret which contains AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and SIGN_KEY"
+        },
+        "request": {
+          "type": "string",
+          "description": "The signing pipeline name to handle the request"
+        },
+        "configMapName": {
+          "type": "string",
+          "description": "The configmap that exists on the cluster for signing of the images e.g. hacbs-signing-pipeline-config-redhatbeta2"
+        },
+        "pipelineImage": {
+          "type": "string",
+          "description": ""
```

Comment:
The signing pipeline says: _"An image with CLI tools needed for the signing"_.  So maybe:
"An image with CLI tools needed for the signing by the internal hacbs-signing-pipeline".

Reply:
Update - although it might also be different due to the fact now we have 2 different signing pipelines (just noticed now) - `hacbs-signing-pipeline` and `blob-signing-pipeline`  - so the image might differ depending on each one we defined in the `request` field. So maybe the text would be:
_"An image with CLI tools needed for the signing by the internal signing pipelines"_

Reply:
Gone with `"An image with CLI tools needed for the signing by the internal signing pipelines"`Thank you 👍🏻